nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-cluster/src/main/java/org/apache/nifi/cluster/coordination/heartbeat/ClusterProtocolHeartbeatMonitor.java

```diff
@@ -91,7 +92,7 @@ public ClusterProtocolHeartbeatMonitor(final ClusterCoordinator clusterCoordinat
         this.clusterNodesPath = zkClientConfig.resolvePath("cluster/nodes");
 
         String hostname = properties.getProperty(NiFiProperties.CLUSTER_NODE_ADDRESS);
-        if (hostname == null) {
+        if (StringUtils.isEmpty(hostname)) {
```

Comment:
This change will conflict with @markap14 change in this PR: https://github.com/apache/nifi/pull/688/files